### PR TITLE
Mu8/santiago/issue4 create domain module

### PR DESCRIPTION
**Pull request by SalosV**
**Create Domain Module:** Issue #4 .

- Add create domain module.

**Testing Done:**

* On real device: Huawei 20P Lite.

**How to test:**

*Create dummy class in the domain module
*Add implementation project (": domain") in the gradle app.
*In the App module (presentation) of the MainActivity.kt class add an instance of the Domain dummy class and verify that the class is imported correctly.

**Signed-off-by:**

* [SalosV](https://github.com/SalosV) + santiagolozada95@gmail.com

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -24,8 +24,10 @@ android {
 }
 
 dependencies {
+
```
Comment (aliceresponde) [inline app/build.gradle]: remove line

File: app/build.gradle [hunk 2]
```diff
@@ -24,8 +24,10 @@ android {
...
+    implementation project(":domain")
     implementation fileTree(dir: 'libs', include: ['*.jar'])
```
Comment (aliceresponde) [inline app/build.gradle:31]: put it, at the beginning of dependencies block

**Review by aliceresponde — CHANGES_REQUESTED** [review]
let me know when the changes were updated.

File: domain/src/main/java/com/example/domain/DummyClass.kt [hunk 3]
```diff
@@ -0,0 +1,3 @@
+package com.example.domain
+
+class DummyClass()
```
Comment (recaldev) [inline domain/src/main/java/com/example/domain/DummyClass.kt]: Please remove this class and instead indicate on the PR description how we can test the integration.

**Review by recaldev — CHANGES_REQUESTED** [review]
A comment added to solve

File: domain/src/main/java/com/example/domain/DummyClass.kt (hunk 3 above)
Comment (SalosV) [reply domain/src/main/java/com/example/domain/DummyClass.kt]: Done

**Comment by SalosV** [comment]
review the changes of the PR.

@recaldev 
@aliceresponde

File: app/build.gradle (hunk 2 above)
Comment (aliceresponde) [reply app/build.gradle]: please sort dependencies 
 
`implementation fileTree(dir: 'libs', include: ['*.jar'])`
`  implementation project(":domain")`

**Review by aliceresponde — APPROVED** [review]

File: app/build.gradle (hunk 2 above)
Comment (SalosV) [reply app/build.gradle:31]: Done

**Review by recaldev — APPROVED** [review]

**Pull request merged**